playwright/Customizations/Compliance.spec.ts

```diff
@@ -83,3 +89,108 @@ test('Create a blueprint with Compliance policy selected', async ({
     ).toBeVisible();
   });
 });
+
+test('Compliance alerts - lint warnings display', async ({ page, cleanup }) => {
+  test.skip(!isHosted(), 'Compliance alerts are not available in the plugin');
+  const blueprintName = 'test-compliance-' + uuidv4();
+  const policyName = 'test-policy-' + uuidv4();
+  const policyType =
+    'Centro Criptológico Nacional (CCN) - STIC for Red Hat Enterprise Linux 9 - Intermediate';
+
+  await cleanup.add(() => deleteBlueprint(page, blueprintName));
+  await cleanup.add(() => deleteCompliancePolicy(page, policyName));
+
+  await ensureAuthenticated(page);
+
+  await createCompliancePolicy(page, policyName, policyType);
+
+  await test.step('Wait for policy to be available', async () => {
+    await navigateToCompliancePolicy(page, policyName);
+    // Wait a bit longer for the policy to be fully available after creation
+    await expect(page.getByRole('row', { name: policyName })).toBeVisible({
+      timeout: 60000,
+    });
+  });
+
+  if (isHosted()) {
+    await page.goto('/insights/image-builder/imagewizard?release=rhel9');
+  } else {
+    await page.goto('/imageWizard?release=rhel9');
+  }
```

Comment:
I am not sure that I understand this. Would this not fail on different OSes? I must admit that testing for compliance in the cockpit might be a bit more complicated. Since this PR is big enough, I suggest creating a separate ticket for testing the feature for cockpit, and skipping tests for it for now. What do you think?